app/src/androidTest/java/com/sdp/movemeet/view/home/LoginActivityTest.java

```diff
@@ -70,7 +70,6 @@
     }
 
     @Test
-    @LargeTest
```

Comment:
Do you know what it could change to use the @Test annotation instead of the @LargeTest annotation ?

Reply:
The first use was to increase the time limit of the test, the "normal" test has a smaller time limit than the large test. If I remember correctly, these annotations can be "small", "medium" or "large" and this can also have an impact on some network access. For example, I believe that a "small test" does not allow access to databases while a medium or large one does.

Reply:
Ok ok, thank you for those details, I wasn't sure!